apps/examples/testcase/ta_tc/mqtt/utc/Kconfig

```diff
@@ -0,0 +1,27 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_TESTCASE_MQTT_UTC
+	bool "MQTT UTC TestCase Example"
+	depends on NETUTILS_MQTT
+	default n
+	---help---
+		Enable the MQTT TestCase example
+
+if EXAMPLES_TESTCASE_MQTT_UTC
+
+config MQTT_BROKER_ADDR
```

Comment:
This is using only in tc. Let's add *"EXAMPLE_TESTCASE_"* prefix to easily distinguish.

Reply:
done